### PR TITLE
Corrected spelling of word 'area'

### PR DESCRIPTION
**Pull request by pranjalshinde**
I have changed a misspelled word 'arena' to 'area' in the document.

### CODE REVIEW

File: files/en-us/games/anatomy/index.html [hunk 1]
```diff
@@ -10,14 +10,17 @@
 <div>{{GamesSidebar}}</div>
 
 <div class="summary">
-<p><span class="seoSummary">This article looks at the anatomy and workflow of the average video game from a technical point of view, in terms of how the main loop should run. It helps beginners to the modern game development arena understand what is required when building a game and how web standards like JavaScript lend themselves as tools.</span> Experienced game programmers who are new to web development could also benefit, too.</p>
+<p><span class="seoSummary">This article looks at the anatomy and workflow of the average video game from a technical point of view, in terms of how the main loop should run. It helps beginners to the modern game development area understand what is required when building a game and how web standards like JavaScript lend themselves as tools.</span> Experienced game programmers who are new to web development could also benefit, too.</p>
```
Comment (chrisdavidmills) [inline files/en-us/games/anatomy/index.html:13]: This change is fine...

File: files/en-us/games/anatomy/index.html [hunk 2]
```diff
@@ -10,14 +10,17 @@
...
 </div>
 
 <h2 id="Present_accept_interpret_calculate_repeat">Present, accept, interpret, calculate, repeat</h2>
 
 <p>The goal of every video game is to <strong>present</strong> the user(s) with a situation, <strong>accept</strong> their input, <strong>interpret</strong> those signals into actions, and <strong>calculate</strong> a new situation resulting from those acts. Games are constantly looping through these stages, over and over, until some end condition occurs (such as winning, losing, or exiting to go to bed). Not surprisingly, this pattern corresponds to how a game engine is programmed.</p>
 
 <p>The specifics depend on the game.</p>
+<p>The goal of every video game is to <strong>present</strong> the user(s) with a situation, <strong>accept</strong> their input, <strong>interpret</strong> those signals into actions, and <strong>calculate</strong> a new situation resulting from those acts. Games are constantly looping through these stages, over and over, until some end condition occurs (such as winning, losing, or exiting to go to bed). Not surprisingly, this pattern corresponds to how a game engine is programmed.</p>
```
Comment (chrisdavidmills) [inline files/en-us/games/anatomy/index.html:21]: Whereas these three lines need to be removed.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hi @pranjalshinde !

First change looks fine, but the second one looks like a mistake?

File: files/en-us/games/anatomy/index.html (hunk 2 above)
Comment (pranjalshinde) [reply files/en-us/games/anatomy/index.html]: Thank you for reviewing the code! It was a mistake which has resolved now.

File: files/en-us/games/anatomy/index.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
 <div>{{GamesSidebar}}</div>
 
 <div class="summary">
-<p><span class="seoSummary">This article looks at the anatomy and workflow of the average video game from a technical point of view, in terms of how the main loop should run. It helps beginners to the modern game development arena understand what is required when building a game and how web standards like JavaScript lend themselves as tools.</span> Experienced game programmers who are new to web development could also benefit, too.</p>
```
Comment (chrisdavidmills) [inline files/en-us/games/anatomy/index.html:13]: My apologies. On giving this a second read, I can see that "arena" is more correct than "area" in this context. But even better would be "...beginners to modern game development understand..."

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
ooop, one more comment, sorry!

File: files/en-us/games/anatomy/index.html (hunk 3 above)
Comment (pranjalshinde) [reply files/en-us/games/anatomy/index.html:13]: Ohh okay!! I will make changes as you said...Thank you!!

**Comment by pranjalshinde** [comment]
Changed "It helps beginners to the modern game development arena understand..." to  "It helps beginners to modern game development understand..."

**Review by chrisdavidmills — APPROVED** [review]
@pranjalshinde this is perfect now, thanks so much for your contribution!

**Comment by pranjalshinde** [comment]
This was my first PR!! Thank you so much for the help!!!

**Comment by chrisdavidmills** [comment]
@pranjalshinde I didn't realize that. Congratulations, and welcome! It's great to have you in the community.

**Pull request merged**